readme.md

```diff
@@ -901,6 +901,7 @@ It's the [Andromeda galaxy](https://simple.wikipedia.org/wiki/Andromeda_galaxy).
 - [When to use `t.plan()`](docs/recipes/when-to-use-plan.md)
 - [Browser testing](docs/recipes/browser-testing.md)
 - [TypeScript](docs/recipes/typescript.md)
+- [Configuring Babel](docs/recipes/babelrc.md)
```

Comment:
Should also add a link in the [ES2015 section](https://github.com/sindresorhus/ava/blob/3120ae6e0f2fad7244426faee93f548ec203a4e8/readme.md#es2015-support).